tslint.json

```diff
@@ -7,6 +7,6 @@
     "member-ordering": false,
     "no-string-based-set-timeout": false,
     "semicolon": [true, "always", "ignore-bound-class-methods"],
-    "no-floating-promises": false
+    "no-floating-promises": true
```

Comment:
Shouldn't need to set this to true. The whole line (with the false) can be removed and it will default to true.

CC: @allileong

Reply:
Just verified this. I'll remove that line in the next commit.